tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -70,6 +70,180 @@ async def test_delayed_assignment_still_errors(dut):
         dut.stream_in_int <= []
 
 
+async def int_values_test(signal, values):
+    """
+    Test integer access to a signal
+    """
```

Comment:
Nit
```suggestion
    """Test integer access to a signal."""
```
(from PEP 257)

Reply:
OK, done.